docs/details/commands/pmm-admin.md

```diff
@@ -364,6 +407,10 @@ If `--stats-collections=db1,db2.col1` then the collectors are run as follows:
 |db1| All the collections|
 |db2| **Only** for collection col1|
 
+Enable all collectors and limit monitoring for the `dbstats` and `indexstats` to all collections in `db1` and for `col1` collection in `db2`, no limit for number of collection in `db1`:
```

Comment:
Enable all collectors and limit monitoring for `dbstats` and `indexstats` to all collections in `db1` and `col1` collection in `db2`, without limiting max-collections-limit for a number of collections in `db1`: